.github/workflows/verify.yaml

```diff
@@ -25,6 +19,26 @@ jobs:
           checkstyle_version: '9.0'
           reporter: 'github-check'
           filter_mode: 'nofilter'
+          properties_file: resources/edc-checkstyle-ignore-enhanced.properties
+
+      - name: Run Checkstyle with additional checks for changed lines
```

Comment:
I have a few questions for better understanding:
1. With this check all Checkstyle rules will be applied again but only on changed files?
2. The severity of RegexpHeader in this check will be default - warning? Which means it doesn't fail the build?

Reply:
1. Actually, applied again on all files. Then reviewdog will filter by changed files